ghost/staff-service/lib/email-templates/partials/commonStyles.hbs

```diff
@@ -0,0 +1,99 @@
+<style>
```

Comment:
We don't use camel case filenames, the file name here can be `common-styles.hbs` or just `styles.hbs`

Reply:
Fixed :)